config/hero.yaml

```
@@ -21,10 +21,10 @@ shortcuts:
     button:
       href: /contributing
       text: Contribute to the project
-  - title: Want to use AEPs for your organization?
+  - title: Want to use AEPs in your organization?
     description: |
-      AEPs are designed to be useful outside of Google. Take a look at how you
-      might choose which AEPs are best suited to your API design needs.
+      AEPs were adapted from [Google's AIP project](https://google.aip.dev/), 
+      and offer benefits and tooling that result in better designed APIs.
```

```suggestion
      AEPs were adapted from [Google's AIP project](https://google.aip.dev/), but also extend it with
      tooling to easily generate and validate that result in better designed APIs, as well as provide clients to 
      consume them.
```

Would love to plug the clients here again.